radis/lbl/loader.py

```diff
@@ -627,6 +627,67 @@ class DatabankLoader(object):
     :class:`~radis.lbl.factory.SpectrumFactory`
     """
 
+    # hardcode attribute names, to prevent typos and the declaration of unwanted parameters
```

Comment:
added to prevent future problems like https://github.com/radis/radis/pull/604/files/f0f7faac8925d3f22a0a4fc1d0c4f8109ea0a014..fa2f3e5d0ad013c9b44bda7dc3a46e2821dbc425#r1299448911

Reply:
> to prevent typos/unwanted syntaxes
How do I add new attributes? We now have an error with interactive plotting because it expects a sf.interactive_params dict member

Reply:
Done